com.regnosys.rosetta/model/RosettaExpression.xcore

```diff
@@ -52,17 +66,50 @@ class RosettaIntLiteral extends RosettaLiteral {
 	}
 }
 
-class ListLiteral extends RosettaLiteral {
+/*
+ * List construction
+ */
+
+class ListLiteral extends RosettaExpression {
```

Comment:
Throughout the code there is a confusion between "literals" and "constants"... A `ListLiteral` is definitely not constant, so I removed it from the `RosettaLiteral` hierarchy. Perhaps we should rename `RosettaLiteral` to `RosettaConstant`? Not sure about the terminology.

Reply:
RosettaPrimitive perhaps?